packages/webpack-rtl-plugin/package.json

```diff
@@ -1,38 +1,36 @@
 {
-	"name": "webpack-rtl-plugin",
+	"name": "@automattic/webpack-rtl-plugin",
 	"version": "2.0.0",
-	"description": "Webpack plugin to produce a rtl css bundle",
+	"description": "Webpack plugin to produce a rtl css bundle. Forked from https://github.com/romainberger/webpack-rtl-plugin",
```

Comment:
Personal opinion: I'd rather keep the description short and one sentence. We can credit the original author somewhere else in a better way.

Reply:
Fixed